docs/area-owners.md

```diff
@@ -2,22 +2,22 @@ The below table lists all the `area-`labels used in the `aspnetcore` repository
 
 | Area label | Owner | Description|
 |--- | ---| --- |
-| area-azure | anurse | |
+| area-azure | BrennanConroy | |
 | area-blazor | mkArtakMSFT | Blazor server and Blazor WASM related |
-| area-commandlinetools | anurse, mkArtakMSFT | dev certs, dotnet watch,  |
-| area-dataprotection | anurse | |
+| area-commandlinetools |  mkArtakMSFT | dev certs, dotnet watch,  |
+| area-dataprotection | BrennanConroy | |
 | area-grpc | shirhatti | |
-| area-healthchecks | rynowak | |
-| area-hosting | anurse | |
-| area-httpclientfactory | anurse | |
+| area-healthchecks | mkArtakMSFT | |
+| area-hosting | BrennanConroy | |
+| area-httpclientfactory | mkArtakMSFT | |
```

Comment:
We've been looking at this one, also I think we should remove it. The only part we own is Polly integration which is in the Extensions repo, and Runtime has the rest.

Reply:
```suggestion
```